austation/code/modules/client/verbs/ooc.dm

```diff
@@ -0,0 +1,6 @@
+/client/ooc(msg as text)
+	. = ..()
+	if(holder?.fakekey)
+		send2chat("**[holder.fakekey]:** [msg2url(msg)]", "ooc")
+	else
+		send2chat("**[key]:** [msg2url(msg)]", "ooc")
```

Comment:
I wonder if I can make a byond account with the username @\here or @\everyone

Reply:
![image](https://user-images.githubusercontent.com/11001588/102706283-971a6c00-42e4-11eb-8e5c-e2d38aeb7b15.png)
most unfortunate

Reply:
h a